runner/config/default.js

```diff
@@ -125,4 +125,7 @@ module.exports = {
   logPrettyPrint: true,
   logRedactPaths: ["req.headers['x-forwarded-for']"], // You should check your privacy policy before disabling this. Check https://getpino.io/#/docs/redaction on how to configure redaction paths
   savePerPage: true, // For activation of the save per page feature
+
+  awsBucketName: "paas-s3-broker-prod-lon-443b9fc2-55ff-4c2f-9ac3-d3ebfb18ef5a", // For uploading files to a aws bucket
```

Comment:
Should we be reading the bucketname and region from config instead of hardcoding?